src/components/Widget/index.tsx

```diff
@@ -136,7 +137,7 @@ export default function Widget({ token, onTokenChange, onReviewSwapClick }: Widg
   }
 
   return (
-    <>
+    <GlowEffect chainId={chainId}>
```

Comment:
GlowEffect should handle getting the chainId itself, not need it to be passed in

Reply:
Initially I did it this way, but then, I figured that making it more explicit gives us a bit more control on it, e.g:
- TokenTable needs different `chainId` than Wallet
- All NFT pages do not pass `chainId` because it has a network default glow as per design
Making it "intelligent" that is resolving `chainId` internally would mean we need some conditionals, which we avoid with this approach.

Reply:
Also, there’s no way to automatically get chainId if we’re using style (from css), not a component. And we’re using style in 99% cases because it’s opaque and doesn’t mess with the layout